test/scripts/same_num_io.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# set -eu
+
+if [ ! -d $k4MarlinWrapper_tests_DIR/inputFiles/ ]; then
+  mkdir $k4MarlinWrapper_tests_DIR/inputFiles
+fi
+
+if [ ! -f $k4MarlinWrapper_tests_DIR/inputFiles/muons.slcio ]; then
+  wget https://github.com/AIDASoft/DD4hep/raw/master/DDTest/inputFiles/muons.slcio -P $k4MarlinWrapper_tests_DIR/inputFiles/
+fi
+
+../run k4run $k4MarlinWrapper_tests_DIR/gaudi_opts/same_num_io.py
+
+input_num_events=$(lcio_event_counter $k4MarlinWrapper_tests_DIR/inputFiles/muons.slcio)
+output_num_events=$(lcio_event_counter Output_DST.slcio)
+
+if [ "$input_num_events" == "$output_num_events" ] || [ "$input_num_events" == 10 ]; then
```

Comment:
```suggestion
if [ "$input_num_events" == "$output_num_events" ]; then
```

Otherwise we would not get an error even if the two numbers are different as long as the number of input events is 10. As said above, what we should be testing is that cases where we want all events, or more events than are present are handled as expected.

Reply:
my bad, I didn't realize that this way the test was a bit useles. Thanks!